src/settings/RefreshTokenRotation.js

```diff
@@ -75,8 +99,55 @@ const RefreshTokenRotation = () => {
           </dl>
         )}
         <div>
-          <Button onClick={invalidateAT}><FormattedMessage id="ui-developer.rtr.invalidateAT" /></Button>
-          <Button onClick={invalidateRT}><FormattedMessage id="ui-developer.rtr.invalidateRT" /></Button>
+          <Button onClick={invalidateAT}>
+            <FormattedMessage id="ui-developer.rtr.invalidateAT" />
+          </Button>
+          <Button onClick={invalidateRT}>
+            <FormattedMessage id="ui-developer.rtr.invalidateRT" />
+          </Button>
+
```

Comment:
Just purge this. It was effective when rotation was tied to API requests but it's no longer as relevant with rotation happening indepdently via a timer.